freud/box.pyx

```diff
@@ -422,6 +422,84 @@ cdef class Box:
         self.thisptr.center(<vec3[float]*> &l_points[0, 0], Np, l_masses_ptr)
         return vecs
 
+    def compute_distances(self, points, query_points):
+        R"""Calculate distances between two sets of points, using periodic boundaries.
+
+        Args:
+            points (:math:`\left(N, 3, \right)` :class:`numpy.ndarray`):
+                Array of points corresponding to points in a data structure.
+            query_points (:math:`\left(N, 3, \right)` :class:`numpy.ndarray`):
+                Array of query points that corresponds to a set of points.
```

Comment:
```suggestion
                Array of query points of the same length as points.
```
Or, if you switch the order of `points` and `query_points`, swap the definition (e.g. `Array of points of the same length as query_points.`)